### PR TITLE
Added eslint-plugin-import to devDependencies, and added some setting…

### PR DESCRIPTION
**Pull request by jzandag**
…s to .eslintrc.js

Reference to Issue #108

### CODE REVIEW

File: src/parsers/pairs.ts [hunk 1]
```diff
@@ -1,10 +1,5 @@
-import type { CSSProperties } from 'react';
+import type { CSSProperties, RefObject } from 'react';
```
Comment (parkerziegler) [inline src/parsers/pairs.ts]: You can remove the `RefObject` import here, this is why the CI run is failing 😄 

```suggestion
import type { CSSProperties } from 'react';
```

**Comment by parkerziegler** [comment]
Awesome, thanks for this @jzandag! Just one last `import` to fix to get the build green, then LGTM!

**Comment by parkerziegler** [comment]
Thanks again @jzandag!

**Comment by jzandag** [comment]
@parkerziegler No problem! Thanks too for the learning opportunity!

**Pull request merged**